src/de.hpi.swa.graal.squeak/src/de/hpi/swa/graal/squeak/nodes/bytecodes/StoreBytecodes.java

```diff
@@ -22,18 +22,18 @@
 public final class StoreBytecodes {
 
     private abstract static class AbstractStoreIntoAssociationNode extends AbstractStoreIntoNode {
-        protected final long variableIndex;
+        protected final Object literal;
```

Comment:
Literals of a method can potentially change, so I don't think it's a good idea to cache them here.